go.mod

```diff
@@ -1,13 +1,19 @@
 module github.com/erh/viamrtsp
 
-go 1.19
+go 1.21
+
+toolchain go1.21.8
```

Comment:
[info] https://go.dev/doc/toolchain

had not seen this before! is there a reason why we are setting toolchain to 1.21.8?

Reply:
it was automatically set by go 1.21 when I ran `go mod tidy`